dev-tools/cloud/launch_cloud_e2e.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cleanup() {
+  r=$?
+
+  echo "--- Cleaning deployment"
+  make -C ${CLOUD_TESTING_BASE} cloud-clean
+
+  exit $r
+}
+trap cleanup EXIT INT TERM
+
+
+echo "--- Creating deployment"
+make -C ${CLOUD_TESTING_BASE} cloud-deploy
+
+# Fleet server URL is obtained in "test-cloude2e-set" target
+# string returned from cloud-get-fleet-url target contains double quotes that should be removed
+FLEET_SERVER_URL=$(make --no-print-directory -C ${CLOUD_TESTING_BASE} cloud-get-fleet-url | tr -d \")
+echo "Fleet server: \"${FLEET_SERVER_URL}\""
+if [[ "${FLEET_SERVER_URL}" == "" ]]; then
+    message="FLEET_SERVER_URL is empty, cloud e2e tests cannot be executed"
+    if [[ "${CI}" == "true" ]]; then
+        buildkite-agent annotate \
+            "${message}" \
+            --context "ctx-cloude2e-test" \
+            --stype "warning"
```

Comment:
That's right , there is an `exit 0` there.

I didn't want to force failing that step (exit code > 0) because it will fail all the builds since that URL is not returned by terraform currently. I could change it to `error` so it gets more attention and don't forget about it. Would that be ok ? @alexsapran

Reply:
Warning annotation style could be dismissed easily, just updated to be `error` the annotation :+1: 
![warning annotation](https://github.com/elastic/fleet-server/assets/5330827/20809c9c-550f-4452-96ef-1302b23d6bf5)

Reply:
There was a type with the style flag , so it did applied the `default` style instead of the warning:
https://buildkite.com/docs/agent/v3/cli-annotate#annotation-styles

In any case, I'll keep the error style.